src/terminal.ts

```diff
@@ -447,6 +447,21 @@ export class FlatpakManifest {
       await config.update(configName, commandPath)
     }
   }
+
+  async deleteOverridenWorkspaceConfig(
+    section: string,
+    configName: string,
+    command: string
+  ): Promise<void> {
+    const commandPath = path.join(this.buildDir, `${command}.sh`)
+    await this.runInRepo(command, true).save(commandPath)
+    const config = vscode.workspace.getConfiguration(section)
+    var config_commandpath: string | undefined = config.get<string>(configName);
+    if (config_commandpath == commandPath) {
+      await config.update(configName, undefined)
```

Comment:
there's no way to reset a config i suppose?

Reply:
config.update(.., undefined) removes the config value, if you mean that? It then falls back to whatever the global configuration is. And since it only matches on the exact path that the extension would want to set when the override is enabled, the user setting would be left alone in any other case

Reply:
Right, i just wondered if there were some explicit api like config.reset but this is fine i guess